components/coda/actions/copy-doc/copy-doc.mjs

```diff
@@ -0,0 +1,41 @@
+import coda from "../../coda.app.mjs";
```

Comment:
Did you run across any errors when developing the actions? e.g. did you observe any 400 errors from Coda on bad inputs? I like to test out edge cases like that and, if the error Coda returns isn't clear, I sometimes like to return my own. See https://pipedream.com/docs/components/guidelines/#error-handling-and-input-validation for more examples.

It's not relevant for all components, but I wanted to mention it if you think the Coda errors can be improved or if there's better input validation we need to do for users.

Reply:
Issue addressed